commented;modifiers;parameterAmount;loc;comment;code
false;public;0;8;;@Test public void testPredicateAssertHelper() throws Exception {     Exchange exchange = new DefaultExchange(context).     Predicate notNull = PredicateBuilder.isNotNull(constant("foo")).     PredicateAssertHelper.assertMatches(notNull, "foo is not null", exchange).     PredicateAssertHelper.assertMatches(notNull, null, exchange). }
false;public;0;18;;@Test public void testPredicateAssertHelperFailed() throws Exception {     Exchange exchange = new DefaultExchange(context).     Predicate notNull = PredicateBuilder.isNotNull(constant(null)).     try {         PredicateAssertHelper.assertMatches(notNull, "foo is not null", exchange).         fail("Should have thrown exception").     } catch (AssertionError e) {     // expected     }     try {         PredicateAssertHelper.assertMatches(notNull, null, exchange).         fail("Should have thrown exception").     } catch (AssertionError e) {     // expected     } }
